Source/Core/Cartographic.js

```diff
@@ -146,6 +146,22 @@ define([
         return result;
     };
 
+    /**
+     * Returns a Cartesian3 position from a Cartographic input.
+     *
+     * @param {Cartographic} cartographic Input to be converted into a Cartesian3 output.
+     * @param {Ellipsoid} [ellipsoid=Ellipsoid.WGS84] The ellipsoid on which the position lies.
+     * @param {Cartesian3} [result] The object onto which to store the result.
+     * @returns {Cartesian3} The position
+     */
+    Cartographic.toCartesian3 = function(cartographic, ellipsoid, result) {
```

Comment:
One more thing, let's be consistent with `fromCartesian` and call this `toCartesian`. Also keep the wording consistent with the other function, see https://github.com/AnalyticalGraphicsInc/cesium/pull/6163/files#diff-f2664d42919c57ae2aa0b2575097bb56R54 and  https://github.com/AnalyticalGraphicsInc/cesium/pull/6163/files#diff-f2664d42919c57ae2aa0b2575097bb56R82